master/docs/manual/cfg-reporters.rst

```diff
@@ -899,3 +899,41 @@ It uses private token auth, and the token owner is required to have at least rep
     :param string baseURL: the base url of the GitLab host, up to and optionally including the first `/` of the path. Do not include /api/
     :param string verbose: Be more verbose
 
+SyslogStatusPush
+~~~~~~~~~~~~~~~~
+
+.. @cindex SyslogStatusPush
+.. py:class:: buildbot.reporters.syslogreporter.SyslogStatusPush
+
+::
+
+    from buildbot.plugins import reporters
+    import syslog
```

Comment:
Isn't this just a dependency of the module, not something the user of it needs to import?